src/components/Pages/HomeSections/videoSection/VideoSection.tsx

```diff
@@ -1,36 +1,165 @@
+                            allowFullScreen
+                            className={
+                                isVideoPlaying
+                                    ? `${styles.videoPlayer} ${styles.videoImage}`
+                                    : styles.none
+                            }
+                        ></iframe>
+                        <Image
+                            alt="videoImage"
+                            src={videoSectionImage}
+                            quality={100}
+                            className={
+                                isVideoPlaying ? styles.none : styles.videoImage
+                            }
+                            onClick={() => handlePlayVideo()}
+                        />
+                        <div
+                            className={
+                                isVideoPlaying
+                                    ? styles.none
+                                    : styles.textWrapper
+                            }
+                        >
+                            <p className={styles.timerP}>
+                                {currentLang["explained in 90 second"]}
+                            </p>
+                            <h1 className={styles.header}>
+                            {currentLang["How does FreeDi work?"]}
+                            </h1>
+                            <p className={styles.underHeaderP}>
+                            {currentLang["Let us walk you through how our app works"]}
+                            </p>
+                        </div>
+                        <div
+                            className={
+                                isVideoPlaying
+                                    ? styles.none
+                                    : styles.logoWrapper
+                            }
+                        >
+                            <FreediLogoIcon size={"1.9rem"} color={"#fff"} />
+                            <FreediIcon
+                                height="1.9rem"
+                                width="4.1rem"
+                                freeColor="#FFE601"
+                                diColor="#fff"
+                            />
+                        </div>
+                        <Image
+                            alt="videoPeopleImage"
+                            src={videoSectionPeopleImage}
+                            quality={100}
+                            className={styles.videoPeopleImage}
+                        />
+                    </div>
+                </>
+            ) : (
+                <>
+                    <div className={styles.textWrapper}>
+                        <h1 className={styles.header}>{currentLang["How does FreeDi work?"]}</h1>
+                        <p className={styles.underHeaderP}>
+                        {currentLang["Let us walk you through how our app works"]}
+                        </p>
+                    </div>
```

Comment:
Text need to align to the right when switching to hebrew.

Reply:
done